src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return ((a ^ b) & (c ^ d)) | ((a ^ c) & (b ^ d));
```

Comment:
Why have you decided to use & instead of &&, what’s the difference?

Reply:
I used instructions from https://www.youtube.com/watch?v=g_cGuJkB9jo&feature=youtu.be Is it incorrect?

Reply:
Done